packages/select/src/components/select/select.tsx

```diff
@@ -122,8 +122,6 @@ export class Select<T> extends AbstractPureComponent<SelectProps<T>, SelectState
 
     private queryList: QueryList<T> | null = null;
 
-    private previousFocusedElement: HTMLElement | undefined;
```

Comment:
All of this `previousFocusedElement` code does what `Popover`'s `shouldReturnFocusOnClose` already does. Just use that.